content/en/docs/architecture/ci-operator.md

```diff
@@ -533,6 +533,40 @@ The _released_ `4.Y` versions of `ClusterImageSet` [manifests](https://github.co
 a tool which ensures that they points to the latest version.
 It is currently not supported that a test claims by `cluster_claim` a cluster with the version which has not been released yet.
 
+### Testing with a Cluster from HyperShift
+[HyperShift](https://hypershift-docs.netlify.app/) is now another supported method to create clusters in OpenShift CI. It is suitable for tests requires only a running cluster, for example, Optional Operator. Tests with modified OCP component are usually not suitable to use HyperShift.
+
+Our [hypershift-hostedcluster-workflow](https://steps.ci.openshift.org/workflow/hypershift-hostedcluster-workflow) enables tests to claim a hosted cluster from DPTP-managed HyperShift (checkout [FAQ]( {{< ref "docs/how-tos/migrating-to-hypershift" >}} )).
+
+Below is an example on how to use `hypershift-hostedcluster-workflow`:
+
+```yaml
+...
+- as: e2e-hypershift
+  steps:
+    cluster_profile: aws-2
+    env:
+      HYPERSHIFT_BASE_DOMAIN: hypershift.aws-2.ci.openshift.org
+      HYPERSHIFT_HC_RELEASE_IMAGE: quay.io/openshift-release-dev/ocp-release@sha256:9ffb17b909a4fdef5324ba45ec6dd282985dd49d25b933ea401873183ef20bf8
+    workflow: hypershift-hostedcluster-workflow
+```
+
+This workflow requires the following environment variables:
```

Comment:
The user needs to change `HYPERSHIFT_BASE_DOMAIN` based on their cluster profile.
The release images are standard release images from quay, the same one when we upgrading a cluster with `--to-image`. I will add some instruction here.

Reply:
It sounds more likely we need a section/page on "HOWTO use your own cluster_profile in the HyperShift workflow"

Reply:
This is not for user to onboard a fresh repo. We are targeting users who already using `ipi-aws` workflow. Thus, they suppose to already know how cluster profile works, since they are already using it. All env they need to set in `hypershift-hostedcluster-workflow` have counterparts in `ipi-aws`, there are nothing new.

Reply:
> We are targeting users who already using ipi-aws workflow.

I would not assume that.
A new repo owner could read this when he wants to do an e2e test without knowing anything about cluster_profile, or ipi-aws workflow.